_studio/shared/src/mfx_static_assert_structs.cpp

```diff
@@ -822,6 +822,13 @@
 #if (MFX_VERSION >= 1026)
         MSDK_STATIC_ASSERT_STRUCT_OFFSET(mfxExtCodingOption3                ,ExtBrcAdaptiveLTR             ,184  )
 #endif
+#if (MFX_VERSION >= 1028)
```

Comment:
shouldn't we see MFX_ENABLE_H264_MAX_FRAME_SIZE here? If not, use MFX_VERSION_NEXT.